sympy/concrete/summations.py

```diff
@@ -259,10 +259,49 @@ def _eval_summation(self, f, x):
 
     def is_convergent(self):
         """
-        Convergence tests are used for checking the convergence of
-        a series. There are various tests employed to check the convergence,
-        returns true if convergent and false if divergent and NotImplementedError
-        if can not be checked. Like divergence test, root test, integral test,
+        We divide the study of convergence of infinite sums and products in
+        two parts.
+
+        First Part:
+        One part is the question whether all the terms are well defined, i.e.,
+        they are finite in a sum and also non-zero in a product. Zero
+        is the analogy of (minus) infinity in products as:
+
+        .. math::
+
+            e^{-\infty} = 0
```

Comment:
This is so short that it could be inlined: ``:math:`e^{-\infty}` `` (if the Sphinx docs are to be trusted). (The colon in as: can be dropped.)